### PR TITLE
feat(playground): Server for AskScript Playground

### PR DESCRIPTION
**Pull request by czerwinskilukasz1**

### CODE REVIEW

**Comment by czerwinskilukasz1** [comment]
@mhagmajer Done

File: src/demoAskServer.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import { parse as parseAskScript } from './askscript';
+
+import chalk = require('chalk');
+import cors = require('cors');
+import { customAlphabet } from 'nanoid';
+
+const nanoid = customAlphabet('1234567890abcdef', 8);
+
+const values = {
+  clientNames: ['a', 'b', 'c', 'd', 'r', 'f', 'g'],
```
Comment (km4) [inline src/demoAskServer.ts:17]: you can change 'r' to 'e' on the array alphabet

File: src/demoAskServer.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  const code = data;
+
+  let askCode;
+  let askCodeSource;
+
+  const id = nanoid();
+
+  try {
+    console.log(id + ' -- ' + new Date().toString());
+    console.log(id + ' -- ' + chalk.grey(`➡️ ${code}`));
```
[the remark this answers is not in the document]
Comment (km4) [reply src/demoAskServer.ts:108]: In my opinion console logs should be deleted on code. If you debugging code its is the best way but on prod add mess.

File: src/demoAskServer.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    // res.json({ error: e.toString() });
+    res.json({
+      data: `document.write('<pre style="color:red; white-space: pre-wrap;">' + ${JSON.stringify(
+        e.toString(),
+        null,
+        2
+      )} + '</pre>');`,
+      language: 'js',
+    });
```
[the remark this answers is not in the document]
Comment (km4) [reply src/demoAskServer.ts:129]: I agree don't know why on API we get `document.write` on the response in JSON (the same on line 157 ).

**Comment by czerwinskilukasz1** [comment]
I am working on a clean version of the Playground (https://github.com/xFAANG/askql/issues/182), so closing this PR now. Will create a new one with proper code changes.

**Pull request closed without merging**